### DISCUSSION TITLE
Nook DRM removal Issues

### DISCUSSION THREAD

**Discussion by seanshuai06** [noDRM/DeDRM_tools#374] [Q&A]
I either have been having issues with DRM removal from the Nook Windows app to calibre. I have set DeDRM to get the key from the NOOK windows app. However, when I import the book over and try to open it with the reader, I get the This is DRM protected pop-up. I have run calibre in debug and got the below script. It is a manga so unsure if that plays a role in why it will not open.

calibre 6.23  embedded-python: True
Windows-10-10.0.19045-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19045')
Python 3.10.1
Windows: ('10', '10.0.19045', 'SP0', 'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: DeDRM (10, 0, 3) && Obok DeDRM (10, 0, 3)
QPA platform: windows
devicePixelRatio: 1.0
logicalDpi: 96.0 x 96.0
physicalDpi: 92.89142857142856 x 93.78461538461538
ApplicationPaletteChange event ignored
Using calibre Qt style: True
ApplicationPaletteChange event received
[0.00] Starting up...
[0.00] Showing splash screen...
[0.16] splash screen shown
[0.16] Initializing db...
[0.16] db initialized
[0.16] Constructing main UI...
DEBUG:    0.0 obok::utilities.py - loading translations
DEBUG:    0.0 obok::dialogs.py - loading translations
DEBUG:    0.0 obok::config.py - loading translations
DEBUG:    0.0 obok::action_err.py - loading translations
[0.56] main UI initialized...
[0.56] Hiding splash screen
Starting QuickView
[0.84] splash screen hidden
[0.84] Started up in 0.84 seconds with 7 books
stdout+stderr from file dialog helper: [b'', b'']
piped data from file dialog helper: [b'\x8f\xd7$\xd8\x02\xb0\xe81\xfb9\x02\xd6\xb2\x86\xbcg\xd2\rd\xf2q_b\xad\xb3p5\xcc\xdcv`m', b'C:\\Users\\Sean Shuai\\AppData\\Local\\Packages\\BarnesNoble.Nook_ahnzqzva31enc\\LocalState\\9781975302016.epub']
DeDRM v10.0.3: Trying to decrypt 9781975302016.epub
DeDRM v10.0.3: Verifying zip archive integrity
FontDecrypt: Checking OPS/pub.opf for IETF font obfuscation keys ... found 'urn:uuid:9d18d049-5459-47d9-9074-e099aaca8186' (PrimaryID)
FontDecrypt: Checking OPS/pub.opf for Adobe font obfuscation keys ... found '9d18d049545947d99074e099aaca8186'
FontDecrypt: There's remaining entries in encryption.xml, adding file ...
DeDRM v10.0.3: Post-processing took 1.1 seconds
DeDRM v10.0.3: Finished after 1.2 seconds
Added Interspecies Reviewers, Vol. 1 to db in: 0.1
Added 1 books in 1.6 seconds

**Comment by ElleKayEm** [comment]
Looks like the DRM is mostly removed.  The problem is

> FontDecrypt: There's remaining entries in encryption.xml

Nook comics and mangas can be non-standard epubs which don't work outside Nook apps anyway.  If the entire book is full page scans, you can probably pull out the images and make a cbz file which could be converted into other formats if needed.
